tools/respecDocWriter.js

```diff
@@ -248,18 +248,28 @@ function makeConsoleMsgHandler(page) {
    * @return {Void}
    */
   return function handleConsoleMessages(whenToHalt) {
-    page.on("console", async message => {
+    page.on("console", message => {
+      const text = message.text();
```

Comment:
@saschanaz, just want to be clear. This line gives back the correct text for errors and warning, and not "JSHandle@error" (so no need to unwrap anymore, it seems). 

Here a sample of an error:  
<img width="1506" alt="screenshot 2018-05-15 00 41 47" src="https://user-images.githubusercontent.com/870154/40004377-eb916f62-57d8-11e8-848a-f1bddbc823f2.png">
 
Here is a sample of a warning: 
<img width="1253" alt="screenshot 2018-05-15 00 42 18" src="https://user-images.githubusercontent.com/870154/40004397-f26c6490-57d8-11e8-84b3-5bd2e2b8cdc9.png">

Reply:
That's because the specific console message argument is string, but we have several `catch (err) { console.error(err); }` there.

Reply:
Maybe we should remove those `console.error(err)` lines...

Reply:
Ah, my bad - thanks for clarifying. Ok, I’d like to combine them and keep text and will use “args” as “details”.